plugin/vim-gh-line.vim

```diff
@@ -114,6 +114,8 @@ func! s:gh_line(action, force_interactive) range
     if s:Github(remote_url)
       let lineRange = s:GithubLineLange(a:firstline, a:lastline, lineNum)
       let url = s:GithubUrl(remote_url) . action . commit . relative . '#' . lineRange
+    elseif s:GoogleSrc(remote_url)
+      let url = s:GoogleSrcUrl(remote_url) . '/+/' . commit . relative . '#' . lineNum
```

Comment:
Does googlesource support range selection?

Reply:
Unfortunately it does not.